src/fontMetrics.js

```diff
@@ -106,7 +106,10 @@ var metrics = {
             return sigma21ScriptScript;
         }
         throw new Error("Unexpected style size: " + style.size);
-    }
+    },
+
+    // \nulldelimiterspace=1.2pt, pg. 344 TeXbook
+    nulldelimiterspace: 1.2 / ptPerEm
```

Comment:
Does this really belong with the font metrics? I think we're getting the value from `ltplain.dtx`.

Also note that as far as I can tell, it's *always* in `pt`, i.e. always the same size, so you should take the *current* fint size into account when emitting an `em`-based measurement, instead of scaling the space with the font, right?